localization/locale_zhcn.php

```diff
@@ -137,24 +137,24 @@
         'bannedRating'  => "你评级评论的权力已被冻结。", # LANG.tooltip_banned_rating
         'tooManyVotes'  => "你已经达到每日投票上限。请明天再来！", # LANG.tooltip_too_many_votes
         'alreadyReport' => "您已报告。", # LANG.ct_resp_error7
-        'textTooShort'  => "[Your message is too short.]",
-        'cannotComment' => "[You have been banned from writing comments.]",
-        'textLength'    => "[Your comment has %d characters and must have at least %d and at most %d characters.]",
+        'textTooShort'  => "[你的留言太短了。]",
```

Comment:
```suggestion
        'textTooShort'  => "你的留言太短了。",
```
Square brackets usually denote missing translations. They should be removed.

Reply:
fixed